settings.gradle.kts

```diff
@@ -1 +1 @@
-rootProject.name = "watermelon-intellij"
+rootProject.name = "watermelon-context"
```

Comment:
Why?

Reply:
Intellij is a reserved word for most occasions.

Reply:
Ok